### PR TITLE
Disable Swaps on Rinkeby for production

### PR DESCRIPTION
**Pull request by dan437**
## Explanation

We sometimes use Rinkeby to test the Swaps flow, but since the actual swapping doesn't happen on Rinkeby, we should only  do this in the dev env and disable Swaps on Rinkeby for production.

### CODE REVIEW

**Review by georgewrmarshall — DISMISSED** [review]
Tested and swaps flow still works on rinkerby on local in dev and code looks good but please get a second review from someone familiar with the selectors

**Review by darkwing — DISMISSED** [review]
Side note: Why isn't this just an array?

**Comment by dan437** [comment]
> Side note: Why isn't this just an array?

Great suggestion, I've changed it to arrays.

**Review by georgewrmarshall — APPROVED** [review]

**Review by darkwing — APPROVED** [review]

**Pull request merged**
